SciGraph-services/src/main/java/io/scigraph/services/jersey/writers/DelimitedWriter.java

```diff
@@ -29,38 +29,43 @@
 import java.lang.annotation.Annotation;
 import java.lang.reflect.Type;
 import java.util.ArrayList;
+import java.util.Collections;
 import java.util.List;
 
 import javax.ws.rs.core.MediaType;
 import javax.ws.rs.core.MultivaluedMap;
 
+import io.scigraph.owlapi.curies.CurieUtil;
 import org.apache.commons.csv.CSVPrinter;
 
 import com.tinkerpop.blueprints.Graph;
 import com.tinkerpop.blueprints.Vertex;
 
 abstract public class DelimitedWriter extends GraphWriter {
 
-  abstract CSVPrinter getCsvPrinter(Writer writer) throws IOException;
+  protected CurieUtil curieUtil = new CurieUtil(Collections.<String, String>emptyMap());
+  abstract CSVPrinter getCsvPrinter(Writer writer, CurieUtil curieUtil) throws IOException;
 
-  static String getCurieOrIri(Vertex vertex) {
-    return TinkerGraphUtil.getProperty(vertex, CommonProperties.CURIE, String.class).or((String)vertex.getProperty(CommonProperties.IRI));
+  static String getCurieOrIri(Vertex vertex, TinkerGraphUtil tinkerGraphUtil) {
+    return tinkerGraphUtil.getProperty(vertex, CommonProperties.CURIE, String.class).or((String)vertex.getProperty(CommonProperties.IRI));
   }
 
   @Override
   public void writeTo(Graph data, Class<?> type, Type genericType, Annotation[] annotations,
-      MediaType mediaType, MultivaluedMap<String, Object> headers, OutputStream out) throws IOException {
+                      MediaType mediaType, MultivaluedMap<String, Object> headers, OutputStream out)
+          throws IOException {
     try (Writer writer = new OutputStreamWriter(out);
-        CSVPrinter printer = getCsvPrinter(writer)) {
+        CSVPrinter printer = getCsvPrinter(writer, curieUtil)) {
       List<String> header = newArrayList("id", "label", "categories");
       printer.printRecord(header);
       List<String> vals = new ArrayList<>();
+      TinkerGraphUtil tinkerGraphUtil = new TinkerGraphUtil(curieUtil);
       for (Vertex vertex: data.getVertices()) {
         vals.clear();
-        vals.add(getCurieOrIri(vertex));
-        String label = getFirst(TinkerGraphUtil.getProperties(vertex, NodeProperties.LABEL, String.class), null);
+        vals.add(getCurieOrIri(vertex, tinkerGraphUtil));
+        String label = getFirst(tinkerGraphUtil.getProperties(vertex, NodeProperties.LABEL, String.class), null);
         vals.add(label);
-        vals.add(TinkerGraphUtil.getProperties(vertex, Concept.CATEGORY, String.class).toString());
+        vals.add(tinkerGraphUtil.getProperties(vertex, Concept.CATEGORY, String.class).toString());
         printer.printRecord(vals);
       }
     }
```

Comment:
This class looks kinda weird now, I think that the TinkerGraphUtil should outputs CURIEs in any case no?